pkg/validation/options.go

```diff
@@ -281,6 +281,11 @@ func parseProviderInfo(o *options.Options, msgs []string) []string {
 	case *providers.GitLabProvider:
 		p.AllowUnverifiedEmail = o.InsecureOIDCAllowUnverifiedEmail
 		p.Groups = o.GitLabGroup
+		p.Projects = o.GitlabProjects
+		// TODO (@NickMeves): merge groups and projects when calling allowed groups - Refactor to Authorize
```

Comment:
Hey! I didn't make this commit 😝

Reply:
Sure, but I don't want you to miss that out if you were responsible of the `Authorize` refactor. Since I'm taking ownership of this we will consider that @NickMeves == me for one commit or two :wink: